testing/kuttl/scripts/pgbackrest-initialization.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+EXPECTED_STATUS=$1
+
+CLUSTER=${CLUSTER:-default}
+
+REPO_POD=$(kubectl -n ${NAMESPACE} get pod --selector=postgres-operator.crunchydata.com/cluster=${CLUSTER},postgres-operator.crunchydata.com/pgbackrest-dedicated --no-headers -o custom-columns=":metadata.name")
+
+# Grab the `status`` line from `pgbackrest info` (regular output, not json since that output includes unnecessary info)
```

Comment:
:pick: :question: 
```suggestion
# Grab the `status` line from `pgbackrest info` (regular output, not json since that output includes unnecessary info)
```